Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc

```diff
@@ -106,6 +107,9 @@ PrimaryVertexValidation::PrimaryVertexValidation(const edm::ParameterSet& iConfi
     theTrackClusterizer_ = new GapClusterizerInZ(iConfig.getParameter<edm::ParameterSet>("TkClusParameters").getParameter<edm::ParameterSet>("TkGapClusParameters"));
   }else if(clusteringAlgorithm=="DA"){
     theTrackClusterizer_ = new DAClusterizerInZ(iConfig.getParameter<edm::ParameterSet>("TkClusParameters").getParameter<edm::ParameterSet>("TkDAClusParameters"));
+    // provide the vectorized version of the clusterizer, if supported by the build
+  } else if(clusteringAlgorithm=="DA_vect") {
```

Comment:
@mmusich although here you just added a clusteringAlgorithm "DA_vect", instead of replacing the older one "DA", but in next [files](https://github.com/cms-sw/cmssw/pull/18448/files#diff-72d8a78a6ccf3dc6cc176329075a4d3f), (https://github.com/cms-sw/cmssw/pull/18448/files#diff-4273a64cde550805c1c977246a830d75), and (https://github.com/cms-sw/cmssw/pull/18448/files#diff-2761a362051a83b5ca702e8f2c172d5e), you have already replaced DA with DA_vect. since #18330  has not been merged yet so do you want this replacement now? Also, do you plan to remove DA once #18330 will be merged.